app/src/main/java/com/a1ishka/bankaccount/domain/TransactionDao.kt

```diff
@@ -0,0 +1,30 @@
+package com.a1ishka.bankaccount.domain
```

Comment:
dao is a part of Room. So it must be in data package

Reply:
moved